app/uk/gov/hmrc/minorentityidentificationfrontend/services/StorageService.scala

```diff
@@ -43,6 +43,9 @@ class StorageService @Inject()(connector: StorageConnector) {
   def removeOverseasTaxIdentifiers(journeyId: String)(implicit hc: HeaderCarrier): Future[SuccessfullyRemoved.type] =
     connector.removeDataField(journeyId, OverseasKey)
 
+  def storeRegistrationStatus(journeyId: String,  registrationStatus: RegistrationStatus)(implicit hc: HeaderCarrier): Future[SuccessfullyStored.type] =
```

Comment:
I don't think we need to save and retrieve the registration status at this time.